### PR TITLE
fix: Wrong event URL is being shared

### LINKED ISSUE #1832: Wrong Event URL is being shared
Not redirecting to eventyay.com

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #1832

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/event/EventUtils.kt [hunk 1]
```diff
@@ -33,7 +33,7 @@ object EventUtils {
 
     private val timeFormat: DateTimeFormatter = DateTimeFormatter.ofPattern("hh:mm a")
     private val dateFormat: DateTimeFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy")
-    private const val frontendUrl = "https://open-event-frontend-dev.herokuapp.com/e/"
+    private const val frontendUrl = "https://eventyay.com/e/"
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/event/EventUtils.kt]: Make this a build config

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/event/EventUtils.kt]: Updated

File: app/build.gradle [hunk 2]
```diff
@@ -37,11 +37,13 @@ android {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             buildConfigField "String", "DEFAULT_BASE_URL", '"https://api.eventyay.com/v1/"'
+            buildConfigField "String", "FRONTEND_URL", '"https://eventyay.com/e/"'
             buildConfigField "String", "MAPBOX_KEY", '"'+MAPBOX_KEY+'"'
             buildConfigField "String", "STRIPE_API_KEY", '"'+STRIPE_API_TOKEN+'"'
         }
         debug {
             buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
+            buildConfigField "String", "FRONTEND_URL", '"https://open-event-fe.netlify.com/e/"'
```
Comment (iamareebjamal) [inline app/build.gradle]: It's fossasia.github.io/open-event-frontend

Comment (liveHarshit) [reply app/build.gradle]: I take it from the frontend documentation. Will update it. :+1:

Comment (liveHarshit) [reply app/build.gradle]: Done

File: app/build.gradle [hunk 3]
```diff
@@ -37,11 +37,13 @@ android {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             buildConfigField "String", "DEFAULT_BASE_URL", '"https://api.eventyay.com/v1/"'
+            buildConfigField "String", "FRONTEND_URL", '"https://eventyay.com/e/"'
```
Comment (iamareebjamal) [inline app/build.gradle]: e/ shouldn't be there

Comment (liveHarshit) [reply app/build.gradle]: Removed

**Comment by anhanh11001** [comment]
Build is failing

**Comment by liveHarshit** [comment]
> Build is failing
Build successful.

**Pull request merged**